pkg/metrics/metrics_test.go

```diff
@@ -105,17 +103,18 @@ func TestNew(t *testing.T) {
 
 				installPath := t.TempDir() + "/" + installTimeFileName
 				if tc.installTime != "" {
-					if err := storeInstallTime(testAppID, installPath, &installInfo{tc.installTime}); err != nil {
-						t.Fatalf("test setup failed: %s", err.Error())
+					if err := storeInstallInfo(installPath, &InstallInfo{
+						InstallTime: tc.installTime,
+					}); err != nil {
+						t.Fatal(err)
```

Comment:
Is cuddling errors not seen as necessasry in test code?

Reply:
wdym? this error is cuddled.

Reply:
Oops, wrong term. I meant wrapped with an error and description.

Reply:
Ah okay:

- "cuddling" is the `if err := thing; err != nil {}` pattern

- "wrapping" is adding extra context, usually with `%w`

We differ from the internal style guide a bit, but we don't usually add extra context to test failures unless it cannot be inferred from the test name and stacktrace